product_template_tree_first/__manifest__.py

```diff
@@ -32,13 +32,13 @@
         'product',
     ],
     'data': [
-        'action_data.xml',
+        'views/action_views.xml',
```

Comment:
Hi @ValentinoDefelice should be rename to this: product_template_action_views.xml